allennlp/models/archival.py

```diff
@@ -64,13 +69,16 @@ def load_model(archive_file: str, cuda_device: int = -1) -> Model:
     # Load config
     config = Params.from_file(os.path.join(tempdir, _CONFIG_NAME))
 
-    # Instantiate model
-    model = Model.load(config,
+    # Load vocabulary
+    vocab = Vocabulary.from_files(os.path.join(tempdir, "vocabulary"))
```

Comment:
I decided to go ahead and do that, as it allowed me to remove `vocabulary` from the `Archive` namedtuple as redudant